src/utils/index.ts

```diff
@@ -129,23 +130,26 @@ export function getProviderOrSigner(library: Web3Provider, account?: string): We
 }
 
 // account is optional
-export function getContract(address: string, ABI: any, library: Web3Provider, account?: string): Contract {
+export function getContract(address: string, ABI: any, library: Web3Provider, account?: string): Contract | null {
   if (!isAddress(address) || address === AddressZero) {
-    throw Error(`Invalid 'address' parameter '${address}'.`);
+    return null;
   }
```

Comment:
after this change, few files are giving typescript error. run `./node_modules/.bin/tsc`

![Screenshot 2022-08-05 at 12 56 21 PM](https://user-images.githubusercontent.com/6604146/183024935-f2257ddc-af6f-47d8-8c47-50edd0a24b85.png)